### PR TITLE
[DOC-512][yba][2024.2] Group mapping to custom roles

### PR DESCRIPTION
**Pull request by ddhodge**
Group mapping to custom roles

@netlify /v2024.2/yugabyte-platform/administer-yugabyte-platform/oidc-authentication/

### CODE REVIEW

File: docs/content/v2024.2/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (asharma-yb) [inline docs/content/v2024.2/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:1]: >1. Click **Create Mappings** (or, if you have existing mappings, **Edit**) to display the **Create Mapping** dialog.

This is now incorrect. From 2024.2 all group mappings, either builtin or custom will be done on the groups tab. 
This applies to both OIDC and LDAP.

File: docs/content/v2024.2/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md [hunk 1]
```diff
@@ -46,12 +46,8 @@ To configure YugabyteDB Anywhere for OIDC, you need to be signed in as a Super A
 
 ## Use OIDC groups with YugabyteDB Anywhere roles
 
-This feature is {{<tags/feature/ea>}}.
-
 If your OIDC provider is configured with user groups, you can map the groups to [YugabyteDB Anywhere roles](../anywhere-rbac/). Users who are members of these groups can then sign in to YugabyteDB Anywhere without needing to be added to YugabyteDB Anywhere first. Users who are members of multiple groups are assigned the most privileged role.
 
-Currently, groups can only be mapped to [built-in](../anywhere-rbac/#built-in-roles) roles.
-
 Note that, if you use group mapping, you must manage users via your OIDC server. You can't add or change user roles in YugabyteDB Anywhere.
```
Comment (asharma-yb) [inline docs/content/v2024.2/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md:52]: Also worth mentioning here that for every user (except SuperAdmin), the role determined via group mapping will override any role assigned previously.

**Review by asharma-yb — CHANGES_REQUESTED** [review]
Maybe we should also attach images of the groups tab since its a new addition with this release. 
The images present in the doc template can be used.

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**